loopFibonacci.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+
+int main() {
+
+int n, next, i, first = 0, second = 1;
+	printf("Enter K number of the Fibonacci sequence.\n");
+	scanf("%d", &n);
+	
+	for (i = 0; i < n; i++) {
+		if (i <= 1)
```

Comment:
Your base case is slightly off! It makes the first three elements 1 instead of just the first two. Any idea how to fix this?